docs/findings/el9.md

```diff
@@ -0,0 +1,332 @@
+
+# Configure System to Forward All Mail For The Root Account
+
+**Condtionally-valid Finding:**
+
+Forwarding-rules for a system's `root` user account is a wholly enterprise-specific &ndash; or even specific to service-group or individual-system level &ndash; determination. While watchmaker _can_ be used to close this finding (via the `.../el9/RuleById/medium/content_rule_postfix_client_configure_mail_alias` control/handler), it relies on the `ash-linux:lookup:root-mail-dest` Pillar-parameter having a value set. If this value is _not_ set, then watchmaker will not close this finding.
+
+```{eval-rst}
+.. note::
+   ``watchmaker``'s automation-content does not have the capability of ensuring that:
+
+   * The Pillar-parameter's ``ash-linux:lookup:root-mail-dest`` value is set to a valid email destination
+   * Even if the ``ash-linux:lookup:root-mail-dest`` value `is` set to a valid email destination, forwarding to that destination will actually `function`
+
+```
+
+# Ensure Chrony is only configured with the server directive
+
+**Condtionally-valid Finding:**
+
+Setup of the `chrony` time-synchronization system can be very site-specific. In fact, some sites may choose not to set it up, at all, due to having other methods for ensuring that their hosts' time is kept properly-synchronized with an authoritative source. By default, `watchmaker` will make no changes to the configuration of the `chrony` time-synchronization service unless one sets the `ash-linux:lookup:use-ntp` Pillar parameter to `True`. If set to `True`, `watchmaker` will attempt to close this finding:
+
+* If one further defines the `ash-linux:lookup:ntp-servers` Pillar-parameter to a list of NTP servers, `watchmaker` will close the finding by configuring the `chrony` service to use that list of servers
+* If one fails to define the `ash-linux:lookup:ntp-servers` Pillar-parameter `watchmaker` will close the finding by configuring the `chrony` service to a default list of servers (the per-vendor "pool" NTP servers maintained by the [Network Time Protocol (NTP) Project](https://ntp.org))
+
+# Enable SSH Server firewalld Firewall Exception
+
+**Invalid Finding:**
+
+This finding may be triggered if only the `ssh` _ports_ are scanned for. The `watchmaker` hardening routines ensure that a broad-scoped (i.e., "allow from all") firewalld exception is made for the `ssh` _service_. The implementation-difference may be seen by comparing the outputs of `firewall-cmd --list-services`
+
+```
+# firewall-cmd --list-services | sed 's/\s\s*/\n/g' | grep ssh
+ssh
+```
+
+and `firewall-cmd --list-ports`:
+
+```
+# firewall-cmd --list-ports | grep ^22
+22/tcp
+```
+
+Watchmaker's implementation will show up only in the output of the former. Some scanners may only expect the execption to show up in the latter.
```

Comment:
```suggestion
Watchmaker's implementation will show up only in the output of the former. Some scanners may only expect the exception to show up in the latter.
```

Reply:
I _really_ need to link my `vim` to `spell`